src/persist/src/postgres.rs

```diff
@@ -341,14 +341,32 @@ impl Consensus for PostgresConsensus {
     async fn scan(&self, key: &str, from: SeqNo) -> Result<Vec<VersionedData>, ExternalError> {
         let q = "SELECT sequence_number, data FROM consensus
              WHERE shard = $1 AND sequence_number >= $2
-             ORDER BY sequence_number";
-        let rows = {
+             ORDER BY sequence_number ASC LIMIT 200";
```

Comment:
I think we want to instead pass the limit down as a parameter to `scan` so nothing has to collect unbounded state for gc

Reply:
ah sure, and gc can just iterate through in pages up until `new_seqno`